server/controllers/user.js

```diff
@@ -110,7 +110,11 @@ function getAll(_, _res) {
 function severalGetInfos(req, _res) {
   const res = new HttpResponseWrapper(_res);
   const listOfUsers = req.body;
-  if (!Array.isArray(listOfUsers) || listOfUsers.length === 0) {
+  if (
+    !Array.isArray(listOfUsers) ||
+    listOfUsers.length === 0 ||
+    listOfUsers.some((user) => typeof user !== "string")
```

Comment:
Because if we pass objects to the API call, the server tries to put these objects in the mysql query, and obviously this produce a server-side error where it should be a bad usage error.